ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -413,25 +419,59 @@ function ListRules(props) {
       });
 
       setRulesWithScheduledChanges(sortedRules);
+      setRewoundRules([]);
 
-      if (es.data && scheduledEs.data) {
-        const shutoffs = es.data.data.shutoffs.map(shutoff => {
-          const returnedShutoff = clone(shutoff);
-          const sc = scheduledEs.data.data.scheduled_changes.find(
-            ses =>
-              ses.product === shutoff.product && ses.channel === shutoff.channel
-          );
-
-          if (sc) {
-            returnedShutoff.scheduledChange = sc;
+      if (
+        emergencyShutoffsAction.data &&
+        scheduledEmergencyShutoffsAction.data
+      ) {
+        const shutoffs = emergencyShutoffsAction.data.data.shutoffs.map(
+          shutoff => {
+            const returnedShutoff = clone(shutoff);
+            /* eslint-disable-next-line max-len */
+            const sc = scheduledEmergencyShutoffsAction.data.data.scheduled_changes.find(
+              ses =>
+                ses.product === shutoff.product &&
+                ses.channel === shutoff.channel
+            );
+
+            if (sc) {
+              returnedShutoff.scheduledChange = sc;
+            }
+
+            return returnedShutoff;
           }
-
-          return returnedShutoff;
-        });
+        );
 
         setEmergencyShutoffs(shutoffs);
       }
-    });
+    }
+  }, [
+    rules.data,
+    scheduledChanges.data,
+    requiredSignoffs.data,
+    emergencyShutoffsAction.data,
+    scheduledEmergencyShutoffsAction.data,
+  ]);
+
+  useEffect(() => {
+    fetchRules(rewindDate ? rewindDate.getTime() : null);
+  }, [rewindDate]);
+
+  useEffect(() => {
+    Promise.all([
+      fetchScheduledChanges(),
+      fetchRules(),
```

Comment:
question: Is this needed here? It looks to me like we're making a `GET` to `/api/rules` twice upon first page load (once because of this effect, and once because of the above). I suspect this can be removed.

Reply:
No, I checked it out and its causing the rules to be fetched twice. I will remove it.